tools/ci-scripts/check-aws-sdk-services

```diff
@@ -7,9 +7,16 @@
 set -eux
 cd aws-sdk
 
+# Check if the VERBOSE environment variable is set
+if [ -n "$ENABLE_V2_SMOKETESTS" ]; then
+    RUN_V2_SMOKETESTS="--cfg v2SmokeTests"
+else
+    RUN_V2_SMOKETESTS=""
+fi
+
 # Invoking `cargo test` at the root directory implicitly checks for the validity
 # of the top-level `Cargo.toml`
-cargo test --all-features
+cargo test --all-features -- "$RUN_V2_SMOKETESTS"
```

Comment:
**correctness**: I don't see any `--cfg` flag for cargo, this looks like a rustc flag only. We can probably just remove this and integrate running these into that future PR/work.